test/multicluster/minikube_base_multicluster_test.go

```diff
@@ -0,0 +1,109 @@
+package multicluster
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/nuodb/nuodb-helm-charts/v3/test/testlib"
+
+	"github.com/gruntwork-io/terratest/modules/helm"
+	"github.com/gruntwork-io/terratest/modules/random"
+)
+
+func verifyNuoSQL(t *testing.T, namespaceName string, adminPod string, databaseName string) {
+	output, err := testlib.RunSQL(t, namespaceName, adminPod, databaseName, "select * from system.nodes")
+	require.NoError(t, err, output)
+	require.True(t, strings.Contains(output, "Storage"))
+	require.True(t, strings.Contains(output, "Transaction"))
+}
+
+func TestKubernetesBasicMultiCluster(t *testing.T) {
+	if os.Getenv("NUODB_LICENSE") != "ENTERPRISE" && os.Getenv("NUODB_LICENSE_CONTENT") == "" {
+		t.Skip("Cannot test multiple SMs without the Enterprise Edition")
+	}
+	testlib.AwaitTillerUp(t)
```

Comment:
I just noticed that we still have this TillerUp function. But we have switched to Helm3. Does it still do something or is it dead code?

Reply:
You are right, this is not needed as we are running with Helm v3. Actually, the function does nothing if Helm 3 is used. 
It seems that we decided not to remove this code in other tests and that is why I kept it here. I'm happy to remove it if you feel strongly about it.

Reply:
I dont. We should remove it in a separate push from everywhere